pkgs/development/libraries/agda/fla/default.nix

```diff
@@ -0,0 +1,14 @@
+fetchFromGitHub:
```

Comment:
Can you use a similar format to the other Agda libraries? Starting with `{ pkgs,` usually. You can then `inherit (pkgs) fetchFromGitHub` I believe.

Reply:
Done!